### ISSUE TITLE
Error in "Push to Github Pages" part of script

### ISSUE THREAD

**Issue by JCLaHoot** [timovv/notion-website-template#7]
Hello, 

Thanks for building this!

I'm getting the error "Process completed with exit code 1." in the Push to Github Pages part of the script, while initializing a blog for the first time.

The whole error:

```
[1](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:1)
Run git config --global user.name "github-actions[bot]"
[2](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:2)
  git config --global user.name "github-actions[bot]"
[3](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:3)
  git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
[4](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:4)
  
[5](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:5)
  GIT_DEPLOY_DIR=dist/site \
[6](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:6)
  GIT_DEPLOY_BRANCH=gh-pages \
[7](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:7)
  GIT_DEPLOY_REPO="https://***@github.com/JCLaHoot/notion-blog-test.git" ./deploy.sh
[8](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:8)
  shell: /usr/bin/bash -e {0}
[9](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:10)
Deploy directory 'dist/site' does not exist. Aborting.
[10](https://github.com/JCLaHoot/notion-blog-test/actions/runs/3843393291/jobs/6545613215#step:7:11)
Error: Process completed with exit code 1.

```
Is there something that I need to do on my end that isn't covered in the README?

**EDIT:**
I managed to run the script* ONCE* by adding the deploy.sh file to the gh-pages branch. I have to re-copy deploy.sh to the gh-pages branch if I want to update the content again.

**Comment by InakiRaba91** [comment]
I'm afraid the repo is not active anymore. But if you're still looking to fix it, you just need to edit [this line](https://github.com/timovv/notion-website-template/blob/main/.github/workflows/main.yml#L28) to:
`docker run \`

There seems to be an issue when mounting the volume with docker-compose, so all files generated inside the container are not preserved in the runner directory where the volume was mounted.

**Comment by timovv** [comment]
This appears to be the same issue as #3. I implemented the fix as described above in #8. You can apply the same fix yourself or recreate from the template again if you’d like take in the change.

I try and keep an eye on this repo every so often but I do get sidetracked with other things unfortunately, and I never managed to get to the bottom of the flakiness.
